includes/discovery/sensors/pre-cache/enexus.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$pre_cache['enexus_battery_test_result_table'] = snmpwalk_cache_oid($device, 'batteryTestResultTable', [], 'SP2-MIB');
```

Comment:
Move this snmpwalk to both the runtime and percent php files and update it to the new SnmpQuery